site/docs/security/tls.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Encryption and Authentication using TLS
+prev: ../index.html
+next: ../sasl
+---
+
+Apache BookKeeper allows clients and autorecovery daemons to communicate over TLS, although this is not enabled by default.
+
+## Overview
+
+The bookies and clients need their own key and certificate in order to use TLS. The key is used for encryption, while the
```

Comment:
yes. the subsequent sentence explains `key` is for encryption and `certificate` for authentication. is that enough? can you guys suggest a better structure for explaining that?